Automattic/jetpack
# External Sources in the Gutenberg Media Sidebar: Add Tracks Events

Fixes https://github.com/Automattic/dotcom-forge/issues/3194

## Proposed changes:

* Added track events so that we can get an idea of the usage of Pexels and Google Photos.
![Captura de Pantalla 2023-07-27 a las 11 40 55](https://github.com/Automattic/jetpack/assets/1989914/6dd6e05c-636a-48d0-b58d-e166e06e7755)

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?
Yes, every time the user searches images in the Pexels or Google Photos categories of the media inserter, we track this data:
- Source (Pexels or Google Photos).
- Number of results.
- Searched term.

## Testing instructions:






* Go to a Jetpack-connected site.
* Edit or create a new Post.
* Open the Media Sidebar and insert a Pexels image.
* You should see an event being tracked, filter in Chrome's Network tab for t.gif with source pexels and the number of results
* Open the Media Sidebar and insert a Google Photos image.
* You should see an event being tracked, filter in Chrome's Network tab for t.gif with source Google Photos and the number of results.
